pip/pip-281.md

```diff
@@ -43,13 +43,13 @@ Just like the implementation of the current [BatchPushSource](https://github.com
 This PIP is to provide a method for users rather than introducing a new interface.
 
 - So it is forward compatible
-- However, connector using this method are not backward compatible.
-For example, the currently Kafka source connector compiles with version 3.1(include this pip) Pulsar dependencies and uses the `notifyError` method, 
-if it switches back to version 3.0(exclude this pip) Pulsar to compile, it will encounter compile errors.
+- However, connector using this method are not backward compatible. 
+For example, If `Kafka source` connector depends on version v3.0(include this pip feature) of pulsar-io and uses the `notifyError` method, 
+when it switches back to depends on 3.0(not include this pip feature) version of pulsar-io to compile will encounter compile errors.
```

Comment:
```suggestion
- However, connectors using this method are not backward compatible. 
For example, If a Kafka source connector built upon pulsar-io v3.1 (including features introduced in this PIP) and uses the `notifyError` method, 
when it switches back to pulsar-io v3.0 (excluding features introduced in this PIP), it will encounter errors during compilation. 
```

do you intend to mean this?